### PR TITLE
Custom operator source published in quay for devtesting nightly operator image

### PR DESCRIPTION
**Pull request by eguzki**
Simplify testing latest (master) operator from OLM. One command, just deploy operator source file:

```bash
kubectl create -f deploy/olm-catalog/apicast-operatorsource.yaml
```

Manifest bundle has been published in application repo https://quay.io/application/apicastoperatormaster/apicast-operator-master with image url: `quay.io/3scale/apicast-operator:master`

### CODE REVIEW

File: deploy/olm-catalog/apicast-operator/apicast-operator.v0.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: apicast-operator
+                image: quay.io/3scale/apicast-operator:master
+                imagePullPolicy: Always
+                name: apicast-operator
+                resources: {}
+              serviceAccountName: apicast-operator
+      permissions:
```
Comment (miguelsorianod) [inline deploy/olm-catalog/apicast-operator/apicast-operator.v0.0.0.clusterserviceversion.yaml]: Before merging this PR, if possible lets wait until https://github.com/3scale/apicast-operator/pull/20 is merged and then regenerate the CSV file. This way we won't have to open another new PR to update it for that.

**Comment by eguzki** [comment]
> Can we re-add the "test/e2e/main_test.go" file? I intend to use it in PR #21 for the e2e tests

why do you need `test/e2e/main_test.go`?

**Comment by miguelsorianod** [comment]
> > Can we re-add the "test/e2e/main_test.go" file? I intend to use it in PR #21 for the e2e tests
> 
> why do you need `test/e2e/main_test.go`?

It's the main entry point of e2e tests. Initializes the testing framework and starts the e2e tests https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md#main-test

File: deploy/olm-catalog/apicast-operator/apicast-operator.v0.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (eguzki) [reply deploy/olm-catalog/apicast-operator/apicast-operator.v0.0.0.clusterserviceversion.yaml]: done

**Comment by eguzki** [comment]
ready for review @miguelsorianod

**Pull request merged**